### PR TITLE
Moving ClickHouse to Vendor Supported list

### PR DESCRIPTION
**Pull request by guykoh**
## Description & motivation
Moved ClickHouse to Vendor Supported list

## Checklist
If you added new pages (delete if not applicable):
- [x] The page has been added to `website/sidebars.js`
- [x] The new page has a unique filename

### CODE REVIEW

**Review by dataders — APPROVED** [review]
thanks @guykoh (love your handle btw)

File: website/docs/docs/available-adapters.md [hunk 1]
```diff
@@ -33,18 +33,20 @@ In addition to maintaining `dbt-core`, [dbt Labs](https://github.com/dbt-labs) m
...
-| Oracle ([dbt-oracle](https://github.com/oracle/dbt-oracle))        | [Profile Setup](oracle-profile)       | `pip install dbt-oracle`     |
-| Rockset ([dbt-rockset](https://github.com/rockset/dbt-rockset))  | [Profile Setup](rockset-profile) | `pip install dbt-rockset` |
-| SingleStore ([dbt-singlestore](https://github.com/memsql/dbt-singlestore)) | [Profile Setup](singlestore-profile) | `pip install dbt-singlestore` |
-| Starburst & Trino ([dbt-trino](https://github.com/starburstdata/dbt-trino)) | [Profile Setup](trino-profile)  | `pip install dbt-trino` |
-| Teradata ([dbt-teradata](https://github.com/teradata/dbt-teradata)) | [Profile Setup](teradata-profile), [Configuration](teradata-configs) | `pip install dbt-teradata` |
-| TiDB ([dbt-tidb](https://github.com/pingcap/dbt-tidb)) | [Profile Setup](tidb-profile) | `pip install dbt-tidb` |
+| Adapter for                                                                                                   | Documentation                                                                                 | Install from PyPi |
```
Comment (dataders) [inline website/docs/docs/available-adapters.md:37]: re-triggering CI...
```suggestion

| Adapter for                                                                                                   | Documentation                                                                                 | Install from PyPi |
```

File: website/sidebars.js [hunk 2]
```diff
@@ -307,6 +307,7 @@ const sidebarSettings = {
         "reference/resource-configs/materialize-configs",
         "reference/resource-configs/firebolt-configs",
         "reference/resource-configs/teradata-configs",
+        "reference/resource-configs/clickhouse-configs",
         "reference/resource-configs/mindsdb-configs"
         "reference/resource-configs/mssql-configs",
```
Comment (JKarlavige) [inline website/sidebars.js]: @dataders Line 311 here is missing a comma, I believe that is the source of the failed build

File: website/sidebars.js [hunk 3]
```diff
@@ -307,6 +307,7 @@ const sidebarSettings = {
         "reference/resource-configs/materialize-configs",
         "reference/resource-configs/firebolt-configs",
         "reference/resource-configs/teradata-configs",
+        "reference/resource-configs/clickhouse-configs",
         "reference/resource-configs/mindsdb-configs"
```
Comment (dataders) [inline website/sidebars.js]:
```suggestion
        "reference/resource-configs/mindsdb-configs",
```

**Pull request merged**